python/tk_multi_loader/proxymodel_entity.py

```diff
@@ -86,8 +88,29 @@ def setFilterFixedString(self, pattern):
         self._cache_hits = 0
         self._cache = {}
 
-        # call base class
-        return QtGui.QSortFilterProxyModel.setFilterFixedString(self, pattern)
+        if len(pattern) > constants.TREE_SEARCH_TRIGGER_LENGTH:
+            # we have a search filter that is longer than one character.
+            # start filtering. Before we can filter, ensure that the entire
+            # data set is loaded in the tree.
+
+            def __fetch_more_r(index):
+                if self.canFetchMore(index):
+                    self.fetchMore(index)
```

Comment:
Same comment as other PR here. I'm assuming it's fine but this requires `fetchMore` to be synchronous in order for `rowCount` to be reliable right? Are we ok to make that assumption here?